deployment/cdk/opensearch-service-migration/lib/migration-assistance-stack.ts

```diff
@@ -114,13 +114,27 @@ export class MigrationAssistanceStack extends Stack {
             securityGroup: comparatorSQLiteSG
         });
 
+        const replayerOutputSG = new SecurityGroup(this, 'replayerOutputSG', {
+            vpc: props.vpc,
+            allowAllOutbound: false,
+        });
+        replayerOutputSG.addIngressRule(replayerOutputSG, Port.allTraffic());
```

Comment:
Does the replayer actually need to accept ingress traffic? Off the top of my head, it seems like it initiates all network activity (pulling from kafka & replaying requests).

I'm thinking that from a security perspective, keeping our SGs as restricted as possible is a good thing.

Reply:
Would the confusion be reduced if the SG was named replayerOutputFilesystemSG (I'm also trying to check my understanding)?  A comment would be helpful too.

Reply:
Do we need all ports open?